ph5/utilities/tests/test_segd2ph5.py

```diff
@@ -17,6 +35,120 @@ def test_bit_weights(self):
         self.assertAlmostEqual(LSB24, segd2ph5.LSB_MAP[24], places=6)
         self.assertAlmostEqual(LSB36, segd2ph5.LSB_MAP[36], places=6)
 
+    def test_write_arrays(self):
+        arrays = \
+            [{'das/serial_number_s': '3X500', 'channel_number_i': 1,
+              'deploy_time/epoch_l': 1544545576},
+             {'das/serial_number_s': '3X500', 'channel_number_i': 2,
+              'deploy_time/epoch_l': 1502293592},
+             {'das/serial_number_s': '3X500', 'channel_number_i': 1,
+              'deploy_time/epoch_l': 1544545576},
+             {'das/serial_number_s': '3X500', 'channel_number_i': 2,
+              'deploy_time/epoch_l': 1502293592}]
+        array_t = \
+            {1: {'3X500': {1502293592: {1: [arrays[0]], 2: [arrays[1]]},
+                           1546021724: {1: [arrays[2]], 2: [arrays[3]]}}}}
+
+        segd2ph5.write_arrays(array_t)
+        ret = segd2ph5.EX.ph5_g_sorts.ph5_t_array
+
+        i = 0
+        for r in ret.iterrows():
+            for k in arrays[0].keys():
+                self.assertEqual(r[k], arrays[i][k])
+            i += 1
+
+    def test_process_traces(self):
```

Comment:
This test is printing / logging to stdout.
Please catch this as we did in other tests.